src/Model/Configuration.php

```diff
@@ -533,11 +537,11 @@ private function getDefaultName(Request $request)
      */
     private function getDefaultAction(Request $request)
     {
-        if (!preg_match('/(\w+)Action$/', $request->attributes->get('_controller'), $match)) {
+        if (!preg_match('/::(\w+)$/', $request->attributes->get('_controller'), $match)) {
             throw new \RuntimeException('Unable to determine controller name');
         }
 
-        return strtolower(preg_replace('/(?<!^)[A-Z]/', '_$0', $match[1]));
+        return strtolower(preg_replace('/(?<!^)[A-Z]/', '_$0', str_replace('Action', '', $match[1])));
```

Comment:
Only strip "Action" if the string ends with it. Right now it will even strip it from the middle, e.g. "getActionPoints" -> "get_points" which is wrong.